pages/osx/yabai.md

```diff
@@ -0,0 +1,24 @@
+# yabai
+
+> A tiling window manager for macOS based on binary space partitioning.
+> More information: <https://github.com/koekeishiya/yabai>.
+
+- Set the layout to bsp:
+
+`yabai -m config layout {{bsp}}`
+
+- Set the window gap to 10pt:
+
+`yabai -m config window_gap {{10}}`
+
+- Enable opacity:
+
+`yabai -m config window_opacity on`
```

Comment:
on and off should be tokens (`{{}}`)

Reply:
It could be, but only if `enable` was changed to `enable/disable`.

Reply:
Enable/disable would need it to be {{on|off}}. But if it's just enable, then I think it should just be {{on}}